templates/core/terraform/identity.tf

```diff
@@ -44,3 +44,11 @@ resource "azurerm_role_assignment" "cosmos_contributor" {
   principal_id         = azurerm_user_assigned_identity.id.principal_id
 }
 
+resource "azurerm_role_assignment" "eventgrid_data_sender" {
+  scope                = data.azurerm_eventgrid_topic.status_changed.id
+  role_definition_name = "EventGrid Data Sender"
+  principal_id         = azurerm_user_assigned_identity.id.principal_id
+  depends_on = [
+    module.airlock_resources
+  ]
```

Comment:
Can you switch this around and do the assignment inside the airlock module?
You will probably need to pass in the api id...

Reply:
Done